### ISSUE TITLE
Dev Build Windows 2023-06-16 Blocked by windows defender

### ISSUE THREAD

**Issue by DavidRainearJr** [DevilXD/TwitchDropsMiner#231]
The latest dev build is being blocked by windows defender, version 2023-06-16 18:05:02+00:00. Had to switch to v15. Letting you know so you can fix it before v16.

**Comment by DevilXD** [comment]
This is a duplicate of #217.
